Cargo.toml

```
@@ -1,6 +1,7 @@
 [workspace]
 members = [
     "chameleon",
+	"crow",
```

Doesn't match the indentation configuration of 4 spaces that is used otherwise

```suggestion
    "crow",
```